reproschema/tests/data/activities/activity1.jsonld

```diff
@@ -15,18 +15,18 @@
         "en": "Over the last 2 weeks, how often have you been bothered by any of the following problems?",
         "es": "Durante las últimas 2 semanas, ¿con qué frecuencia le han molestado los siguintes problemas?"
     },
-    "messages": [
-      {
-        "message": "Test message: Triggered when item1 value is greater than 1",
-        "jsExpression": "item1 > 1"
-      }
-    ],
     "compute": [
         {
-            "variableName": "activity1_total_score",
-            "jsExpression": "item1 + item2"
+            "variableName": "phq9_total_score",
+            "jsExpression": "phq9_1 + phq9_2 + phq9_3 + phq9_4 + phq9_5 + phq9_6 + phq9_7 + phq9_8 + phq9_9"
```

Comment:
if we use these variable names(`phq9_1`, `phq9_2` , .. ), they need to be defined in addProperties. That is why I used "item1 + item2"